ament_cmake_mypy/cmake/ament_mypy.cmake

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Canonical, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Add a test to statically check Python types using mypy.
+#
+# :param CONFIG_FILE: the name of the config file to use, if any
+# :type CONFIG_FILE: string
+# :param TESTNAME: the name of the test, default: "mypy"
+# :type TESTNAME: string
+# :param ARGN: the files or directories to check
+# :type ARGN: list of strings
+#
+# @public
+#
+function(ament_mypy)
+  cmake_parse_arguments(ARG "" "CONFIG_FILE,TESTNAME" "" ${ARGN})
```

Comment:
A comma is not a valid list separator in CMake. It should be a semicolcon.

Reply:
fixed in 5c6d139